File: jisoo/programmers/level1/K번째수.py
"""
https://school.programmers.co.kr/learn/courses/30/lessons/42748
"""


def solution(array, commands):
  answer = []
  for command in commands:
    start, end, index = command
    temp = array[start - 1:end]
    temp.sort()
    answer.append(temp[index - 1])

  return answer

File: seongwon/LV1/k번째수.js
function solution(array, commands) {
  let result = [];
  for (let i = 0; i < commands.length; i++) {
    let start = commands[i][0] - 1;
    let end = commands[i][1] - 1;
    let num = commands[i][2];

    let temp = array.slice(start, end + 1);
    temp = temp.sort((a, b) => a - b);
    result.push(temp[num - 1]);
  }
  return result;
}
